### PR TITLE
align with OSI template

### PR DESCRIPTION
**Pull request by lsmith77**
## What do these changes do?

Align with the OSI template to hopefully get Github to detect the license properly:
https://twitter.com/Stof70/status/1585287286786330625

## Are there changes in behavior for the user?

Will help anyone that wants to use automation to check license compatibility

## Checklist

N/A

### CODE REVIEW

**Comment by lsmith77** [comment]
then again the issue might be related to https://github.com/licensee/licensee/issues/534#issuecomment-1054040286, ie. reference to the license in the README

**Comment by Dreamsorcerer** [comment]
The copyright year was removed deliberately, as it adds maintenance burden (and apparently, there are some users who think the entire library is unmaintained if the copyright year has not been updated).

If you can play with that licensee library and figure out exactly what the cause is, maybe we can do something else.

**Comment by lsmith77** [comment]
yeah. I am trying to find out more about how Github determines the license and come back.

**Comment by webknjaz** [comment]
> I am trying to find out more about how Github determines the license

It uses `licensee` to determine how similar the found licenses are to their canonical forms. It's a Ruby app. You can install it and call via CLI in project checkouts to learn how GH "sees" them.

File: LICENSE.txt [hunk 1]
```diff
@@ -1,10 +1,10 @@
-   Copyright aio-libs contributors.
+  Copyright 2014-2022 aio-libs contributors.
```
Comment (webknjaz) [inline LICENSE.txt:1]: The Linux Foundation recommends dropping the year or using just the start date. Updating the year is just busy work.

**Pull request closed without merging**